xml/templates/message_templates/payment_or_refund_notification_html.tpl

```diff
@@ -71,45 +69,45 @@
     </tr>
     <tr>
       <td {$labelStyle}>
-  {ts}{if $component eq 'event'}Total Fees{/if}{/ts}
+        {ts}Total Amount{/ts}
       </td>
       <td {$valueStyle}>
-  {$totalAmount|crmMoney}
+        {$totalAmount|crmMoney}
       </td>
       </tr>
       <tr>
       <td {$labelStyle}>
-  {ts}This Payment Amount{/ts}
+        {ts}This Payment Amount{/ts}
       </td>
       <td {$valueStyle}>
-  {$paymentAmount|crmMoney}
+        {$paymentAmount|crmMoney}
       </td>
       </tr>
      <tr>
       <td {$labelStyle}>
-  {ts}Balance Owed{/ts}
+        {ts}Balance Owed{/ts}
       </td>
        <td {$valueStyle}>
-  {$amountOwed|crmMoney}
+         {$amountOwed|crmMoney}
       </td> {* This will be zero after final payment. *}
      </tr>
      <tr> <td {$emptyBlockStyle}></td>
      <td {$emptyBlockValueStyle}></td></tr>
       {if $paymentsComplete}
       <tr>
       <td colspan='2' {$valueStyle}>
-  {ts}Thank-you. This completes your payment for {if $component eq 'event'}{$event.event_title}{/if}.{/ts}
+        {ts}Thank-you. This completes your payment for {if $component eq 'event'}{$event.event_title}{/if}.{/ts}
```

Comment:
@JoeMurray what would be good text in case the payment is not related participant.